### PR TITLE
feat: Axiom staging

### PR DESCRIPTION
**Pull request by bonustrack**
### Summary

This PR is staging branch deployed to https://next.snapshotx.xyz

Check https://github.com/snapshot-labs/sx-monorepo/pull/278 for full feature set.

### CODE REVIEW

File: apps/ui/src/networks/evm/constants.ts [hunk 1]
```diff
@@ -470,5 +475,127 @@ export const EDITOR_EXECUTION_STRATEGIES = [
...
+        }
+      }
+    }
+  },
+  {
+    address: '',
+    type: 'Isokratia',
+    name: EXECUTORS.Isokratia,
+    about:
+      'This strategy enables offchain votes on the space. The validity of votes and voting power is verified onchain in bulk using a zkSNARK, which then triggers the execution of transactions.',
```
Comment (bonustrack) [inline apps/ui/src/networks/evm/constants.ts:543]:
```suggestion
      'This strategy enables offchain votes on the space. The validity of votes is verified onchain in bulk using a zkSNARK, which then triggers the execution of transactions.',
```

File: yarn.lock [hunk 2]
```diff
@@ -2185,6 +2203,31 @@
...
+    "@ethersproject/solidity" "^5.7.0"
+    "@ethersproject/wallet" "^5.7.0"
+    "@openzeppelin/merkle-tree" "^1.0.5"
+    bn.js "^5.2.1"
+    cross-fetch "^3.1.5"
+    micro-starknet "^0.2.3"
+    randombytes "^2.1.0"
+    snake-case "^3.0.4"
+    starknet "5.25.0"
+
```
Comment (bonustrack) [inline yarn.lock:2230]:
```suggestion
```

File: apps/ui/src/networks/evm/constants.ts [hunk 3]
```diff
@@ -488,6 +493,134 @@ export function createConstants(networkId: NetworkID) {
...
+      ): Promise<{ address: string; txId: string }> => {
+        return client.deployAxiomExecution({
+          signer,
+          params: {
+            controller: params.controller || '0x0000000000000000000000000000000000000000',
+            quorum: BigInt(params.quorum),
+            contractAddress: params.contractAddress || '0x0000000000000000000000000000000000000000',
+            slotIndex: BigInt(params.slotIndex),
+            space: spaceAddress,
+            querySchema: formatBytes32String('0')
```
Comment (Sekhmet) [inline apps/ui/src/networks/evm/constants.ts:521]:
```suggestion
            querySchema: '0xa09cc16ccaa32b96ca5c404c1b4be60d7883a7178f432e8f9f3c22157fc0f873'
```

**Comment by Sekhmet** [comment]
@bonustrack this is ready to review.

**Comment by Sekhmet** [comment]
Main PR was merged: https://github.com/snapshot-labs/sx-monorepo/pull/278

**Pull request closed without merging**